docs/bnb_smart_chain/slashing/slash-events.md

```diff
@@ -0,0 +1,50 @@
+# BSC Slashing Events
+
+Three types of malicious behaviors can lead to slashing on the BSC network.
+
+## Double Sign
+
+Anyone can submit a slash request with evidence of Double Sign. The evidence must adhere to the following rules:
+
+* Two block headers have the same height and the same parent block hash
+* Two block headers are sealed by the same validator
+* Two signatures of these two blocks must not be the same
+* The time of these two blocks must be within the validity of the evidence, which is 24 hours
+
+If the evidence is valid:
+
+1. **10,000BNB** would be slashed from the **self-delegated** BNB of the validator
+2. If the self-delegator’s stake amount on the validator is less than 10,000BNB, then the unbonding delegation balance would be slashed if it exists until totally 10,000BNB slashed from the self-delegator of the validator. However, if all the slashed BNB is less than 10,000, all the remaining stake of the self-delegator will be slashed
+3. **1000** of slashed BNB would allocate to the submitter as a reward
+4. The remaining slashed BNB will be allocated to the custody addresses of validators participating in the next distribution
+5. Set the validator ‘jailed’ with a duration of **292 years**, and remove it from the active validator set
+
+
+## Malicious Vote
+Anyone can submit a slash request on BSC with the evidence of Malicious Vote. The evidence must adhere to the following rules:
+
+* The target number voted by two votes lags behind the block header of the canonical chain by no more than 256
+* The source numbers of the two votes are both smaller than their respective target numbers
+* The source hash and target hash of the two votes are both not equal
+* The target number of the two votes is the same or the span of one vote includes the span of the other vote
+* The two votes are signed by the same voting key, and the verification of signatures are both passed
+* The voting key used for signing is in the list sent by the last two breathe blocks
+
+If the evidence is valid:
+
+1. **200BNB** would be slashed from the **self-delegated** BNB of the validator
+2. **5BNB** would allocate to the submitter from the system reward contract as a reward if the validator is active when the evidence submitted
```

Comment:
This is not exactly true.

Reply:
there is a threshold when claiming from the system reward contract.